dnd-bot/dnd_bot/logic/prototype/creature.py

```diff
@@ -17,26 +17,60 @@ def __init__(self, x=0, y=0, sprite: str = '', name: str = 'Creature', hp: int =
             drop_money = []
         if drops is None:
             drops = []
+
         self.hp = hp
         self.max_hp = hp
-        self.strength = strength
-        self.dexterity = dexterity
-        self.intelligence = intelligence
-        self.charisma = charisma
-        self.perception = perception
+        self.base_strength = strength
+        self.base_dexterity = dexterity
+        self.base_intelligence = intelligence
+        self.base_charisma = charisma
+        self.base_perception = perception
+
         self.initiative = initiative
         self.action_points = action_points
-        self.level = level
-        self.equipment = equipment
-        self.drop_money = drop_money
         self.initial_action_points = action_points
+        self.equipment = equipment
+        self.level = level
         self.experience = experience
         self.creature_class = creature_class
+
+        self.drop_money = drop_money
         self.drops = drops
 
         self.ai = ai
         self.move_queue = []
 
+# ----------------------------------------------------- properties -----------------------------------------------------
+    def eq_stats(self, stat):
```

Comment:
Krótka dokumentacja tej funkcji by się przydała jeszcze.

Reply:
bruh, to dosłownie 1 linijka

Reply:
chodzi o to żeby było jasne po co jest ta funkcja